spec.html

```diff
@@ -4041,7 +4041,7 @@ <h1>Well-Known Intrinsic Objects</h1>
               <td>
               </td>
               <td>
-                The prototype of wrapped Iterator objects returned by Iterator.from (<emu-xref href="#sec-%wrapforvaliditeratorprototype%-object"></emu-xref>)
+                The prototype of wrapped iterator objects returned by Iterator.from (<emu-xref href="#sec-%wrapforvaliditeratorprototype%-object"></emu-xref>)
```

Comment:
This use could remain capitalized as well, since the returned values do in fact inherit from `Iterator`.

Reply:
Elsewhere we use "Iterator instance objects" to refer to iterators in a way that also requires that they inherit from `Iterator`.